crosshair/main.py

```diff
@@ -154,7 +154,7 @@ def command_line_parser() -> argparse.ArgumentParser:
             "--analysis_kind",
             type=analysis_kind,
             metavar="KIND",
-            default=(AnalysisKind.PEP316, AnalysisKind.icontract, AnalysisKind.asserts),
+            default=(AnalysisKind.PEP316, AnalysisKind.icontract, AnalysisKind.asserts, AnalysisKind.hypothesis),
```

Comment:
```suggestion
            default=(AnalysisKind.PEP316, AnalysisKind.icontract, AnalysisKind.hypothesis, AnalysisKind.asserts),
```

If a Hypothesis-decorated function starts with leading assertions, we still want to use your new feature rather than treating those asserts as preconditions.

Reply:
Yep, I have some hypothesis tests and stuff that ive messed around with locally, just haven't committed them yet.